### DISCUSSION TITLE
Session access takes time after configuring gdm-settings.

### DISCUSSION THREAD

**Discussion by Tuxman2** [realmazharhussain/gdm-settings#222] [Q&A]
I installed gdm-settings, and configured that for my machine. After entering my password in the login screen, it takes me +/- 7-8 seconds before showing the desktop. Without the use of gdm-settings, access to my desktop is fast (default gdm configuration). Can gdm-settings cause this kind of thing ?

Thanks.

Specifications: 

Fedore 39 workstation: 

- Gnome 45.3
- Kernel 6.6.x

**Comment by realmazharhussain** [comment]
> Can gdm-settings cause this kind of thing ?

Possibly. Did you only apply GDM customizations before delay in session startup started happening or did you apply any other modifications to the system e.g. custom bash profile, autostart apps, systemd services, shell extensions, etc.?

**Reply by Tuxman2** [reply]
I installed some shell extensions but I didn't encounter this long delay before applying GDM customizations. For your information, I use your last stable version from flatpak. It is very strange because I also encounter this problem on SSD.
